src/import/import-figma.ts

```diff
@@ -639,44 +679,58 @@ export const getTextStyles = (
     if (bp) {
       // If the style does correspond to a breakpoint, add the values under that key
       name = name.replace(`/${bp}`, '');
-      setWith(variants, `${name}.fontFamily.${bp}`, values.fontFamily);
-      setWith(variants, `${name}.fontSize.${bp}]`, values.fontSize);
-      setWith(variants, `${name}.fontStyle.${bp}]`, values.fontStyle);
-      setWith(variants, `${name}.fontWeight.${bp}]`, values.fontWeight);
-      setWith(variants, `${name}.letterSpacing.${bp}]`, values.letterSpacing);
-      setWith(variants, `${name}.lineHeight.${bp}]`, values.lineHeight);
+      setWith(variants, `${name}.$value.fontFamily.${bp}`, values.fontFamily);
+      setWith(variants, `${name}.$value.fontSize.${bp}]`, values.fontSize);
+      setWith(variants, `${name}.$value.fontStyle.${bp}]`, values.fontStyle);
+      setWith(variants, `${name}.$value.fontWeight.${bp}]`, values.fontWeight);
       setWith(
         variants,
-        `${name}.textDecorationLine.${bp}]`,
+        `${name}.$value.letterSpacing.${bp}]`,
+        values.letterSpacing
+      );
+      setWith(variants, `${name}.$value.lineHeight.${bp}]`, values.lineHeight);
+      setWith(
+        variants,
+        `${name}.$value.textDecorationLine.${bp}]`,
         values.textDecorationLine
       );
-      setWith(variants, `${name}.textTransform.${bp}]`, values.textTransform);
+      setWith(
+        variants,
+        `${name}.$value.textTransform.${bp}]`,
+        values.textTransform
+      );
       return;
     }
 
     // Otherwise just assign the style object as normal
-    variants[name] = values;
+    setWith(variants, `${name}.$value]`, values);
   });
 
   // Iterate over the styles and flatten any unnecessary responsive values
-  // e.g. change `fontSize: { base: '1rem', sm: '1rem', md: '1rem' }` to just `fontSize: '1rem'`
+  // e.g. change `fontSize: { base: { $value: '1rem' }, sm: { $value: '1rem' }, md: { $value: '1rem' } }` to just `fontSize: { $value: '1rem' }`
   const flatten = (value: string | string[]) => {
     const objValues =
       typeof value === 'object' ? Object.values(value) : undefined;
     return objValues?.every((v) => v === objValues[0]) ? objValues[0] : value;
   };
   Object.keys(variants).forEach((k) => {
-    const v = variants[k];
-    variants[k] = {
-      fontFamily: flatten(v.fontFamily),
-      fontSize: flatten(v.fontSize),
-      fontStyle: flatten(v.fontStyle),
-      fontWeight: flatten(v.fontWeight),
-      letterSpacing: flatten(v.letterSpacing),
-      lineHeight: flatten(v.lineHeight),
-      textDecorationLine: flatten(v.textDecorationLine),
-      textTransform: flatten(v.textTransform),
-    };
+    if (typeof variants[k] !== 'string') {
```

Comment:
As far as I can tell it's not actually possible for `typeof variants[k] !== 'string'` to evaluate to `false`. If that's true can we update the type definition and remove this check?